editor/plugins/tiles/tile_map_layer_editor.cpp

```diff
@@ -4497,13 +4491,12 @@ TileMapLayerEditor::TileMapLayerEditor() {
 	toggle_grid_button->set_toggle_mode(true);
 	toggle_grid_button->set_tooltip_text(TTR("Toggle grid visibility."));
 	toggle_grid_button->connect(SceneStringName(toggled), callable_mp(this, &TileMapLayerEditor::_on_grid_toggled));
-	toggle_grid_button->set_accessibility_name(TTRC("Grid"));
 	tile_map_toolbar->add_child(toggle_grid_button);
 
 	// Advanced settings menu button.
 	advanced_menu_button = memnew(MenuButton);
 	advanced_menu_button->set_flat(false);
-	advanced_menu_button->set_accessibility_name(TTRC("Advanced"));
+	advanced_menu_button->set_accessibility_name(TTRC("Advanced settings."));
```

Comment:
Unsure how this **MenuButton** looks like in context right now, but I'm surprised this is not a tooltip.